test/lib/ufe/testLight.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
```

Comment:
Would have been good to get some undo / redo testing done as well.  Perhaps next pull request.